apis/core/hydra/column-mapping.ttl

```diff
@@ -5,33 +5,70 @@ prefix code:  <https://code.described.at/>
 prefix schema:   <http://schema.org/>
 prefix sh:     <http://www.w3.org/ns/shacl#>
 
-cc:ColumnMapping
+cc:LiteralColumnMapping
   a
     hydra:Class ;
   hydra:supportedOperation
   [
     a hydra:Operation,
       schema:UpdateAction ;
     hydra:title
-      "Update Column Mapping" ;
+      "Edit column mapping" ;
     hydra:method
       "PUT" ;
     hydra:expects
       cc:ColumnMapping,
-      <shape/column-mapping/create> ;
+      <shape/column-mapping/literal> ;
     code:implementedBy [
       a
         code:EcmaScript ;
+      code:link
+        <file:handlers/column-mapping#putLiteral> ;
+    ]
+  ] ,
+  [
+    a hydra:Operation,
+      schema:DeleteAction ;
+    hydra:title
+      "Delete column mapping" ;
+    hydra:method
+      "DELETE" ;
+    code:implementedBy [
+      a
+        code:EcmaScript ;
+      hydra:expects
+        cc:ColumnMapping ;
+      code:link
+        <file:handlers/column-mapping#remove> ;
+    ]
+  ] .
```

Comment:
I think we could have only one DELETE operation supported by a `cc:ColumnMapping` - superclass to the other two